package.json

```diff
@@ -6,27 +6,18 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
-  "license": "ISC",
+  "license": "MIT",
+  "author": "Microsoft Corp.",
   "dependencies": {
-    "az-iot-bi-test": "0.0.2",
-    "az-iot-helper": "^0.1.1",
-    "azure-event-hubs": "0.0.2",
```

Comment:
>azure-event-hubs [](start = 5, length = 16)

This package is required by iot-hub.js .
Please add it back in package.json in az-blink.